tests/test/faaslet/test_mpi.cpp

```diff
@@ -36,6 +36,11 @@ TEST_CASE("Test MPI broadcast", "[wasm]")
     checkMpiFunc("mpi_bcast");
 }
 
+TEST_CASE("Test MPI cartesian coordinates", "[wasm]")
```

Comment:
There are a lot of tests now tagged with `wasm`, perhaps in future we could tag them with something more meaningful or with multiple tags (e.g. tagging these ones with `mpi` so we could do `/build/faasm/bin/tests "[mpi]"`). Not something to worry about now though

Reply:
Will handle this quickly now. I also thought about it.